cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -578,6 +592,9 @@ public void like(String testcase) throws QueryValidationException {
     "*         sortBy name/sort.ascending/string     # Jo Jane; Ka Keller; Lea Long",
     "*         sortBy name/sort.descending           # Lea Long; Ka Keller; Jo Jane",
     "*         sortBy name/sort.descending/string    # Lea Long; Ka Keller; Jo Jane",
+    "*         sortBy alternateEmail                 # Jo Jane; Lea Long; Ka Keller",
+    "*         sortBy alternateEmail/sort.ascending  # Jo Jane; Lea Long; Ka Keller",
+    "*         sortBy alternateEmail/sort.descending # Ka Keller; Jo Jane; Lea Long",
```

Comment:
The correct sort order for alternateEmail/sort.descending is `Ka Keller; Lea Long; Jo Jane`.
Note that sort.descending must be the reverse of sort.ascending because all values are distinct.
```
"fff" + 600 x "f" + "gfffffffffff@example.com"          # Jo Jane
"fff" + 600 x "f" + "hfffffffffff@example.com"          # Lea Long
"ffg" + 600 x "f" + "ffffffffffff@example.com"          # Ka Keller
```

Please fix the code: `ORDER BY left(term,600) DESC, term DESC`